### PR TITLE
Softsign activation function

### PR DESCRIPTION
**Pull request by jpeg729**
I have tried implementing the softsign activation function proposed by (Bergstra et al., 2009).

My net of LSTMs compiles and runs, but during training it outputs ```loss: nan``` and the net doesn't seem to output very much of anything. I'm not sure what is going wrong.

### Formula and motivation

The formula is ```x/(1 + |x|)```

As Glorot says:
> The softsign is similar to the hyperbolic tangent (its range is -1 to 1) but its tails are quadratic polynomials rather than exponentials, i.e., it approaches its asymptotes much slower.

Some say that softsign is better than tanh because it is faster and less prone to saturation.

### CODE REVIEW

File: keras/backend/theano_backend.py [hunk 1]
```diff
@@ -584,6 +584,11 @@ def softplus(x):
     return T.nnet.softplus(x)
 
 
+def softsign(x):
+    # return T.nnet.softmax(x)
```
Comment (fchollet) [inline keras/backend/theano_backend.py]: What about this comment though?

Comment (jpeg729) [inline keras/backend/theano_backend.py]: I saw that for softmax, theano was only used in the 2D case, and not being at all sure of what I am doing I preferred to implement it directly, and leave the comment as a reminder of something that could be optimised later.

Comment (jocicmarko) [inline keras/backend/theano_backend.py]: Shouldn't Theano's **abs ** be used instead of Python's?

Comment (jpeg729) [inline keras/backend/theano_backend.py]: Well spotted, but given that a function called **abs** is defined elsewhere in the file, I thought it would be ok.

I tested it and it is still giving me ```loss: nan```

**Comment by jpeg729** [comment]
I figured out how to import theano's softsign function, and it does seem to be working.

**Comment by fchollet** [comment]
LGTM (sorry it took a while to review)
